omnibus/files/server-ctl-cookbooks/infra-server/templates/default/opensearch_jvm.opts.erb

```diff
@@ -0,0 +1,63 @@
+## JVM Configuration
+
+<% # Wants: log_dir, enable_gc_log, heap_size, new_size, jvm_opts, tmp_dir %>
+
+<% @jvm_opts.each do |opt| %>
+<%=opt%>
+<% end %>
+# Do not override xmx/xms/newsize in
+# jvm_options - instead set the attributes in chef-server.rb
+-Xmx<%=@heap_size%>m
+-Xms<%=@heap_size%>m
+-XX:NewSize=<%= @new_size %>M
+
+# These are defaulted based on what 7.9 ships with.
+# https://github.com/elastic/elasticsearch/blob/7.9/distribution/src/config/jvm.options
```

Comment:
Is there an equivalent opensearch doc we can link to?

Reply:
Currently we don't have any link fromOpenSearch so using the ES-7.9 link.